### PR TITLE
v0.4.0: Support .editorconfig auto-complete

### PR DESCRIPTION
**Pull request by jedmao**
#99 included in this release. Lots of linting issues to fix and dependency updates included as well.

### CODE REVIEW

File: src/transformations/insertFinalNewline.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
```
Comment (SamVerschueren) [inline src/transformations/insertFinalNewline.ts]: Use strict can be removed here like all the other places

**Comment by jedmao** [comment]
I really want to release this, but there are issues on Travis-CI and with my permissions as well. This is all the time I'm going to spend on this today.

**Comment by jedmao** [comment]
It looks like `worbench.files.actions.closeAllFiles` is now `workbench.action.closeAllEditors`.

**Comment by jedmao** [comment]
@SamVerschueren removed in c494fd5.

**Comment by jedmao** [comment]
Published in version `0.4.0`.

**Pull request merged**